charts/prometheus/templates/prometheus-alerts-configmap.yaml

```diff
@@ -30,7 +30,7 @@ data:
         {{- tpl .Values.additionalAlerts.airflow $ | nindent 8 }}
         {{- end }}
         - alert: AirflowDeploymentUnhealthy
-          expr: sum by(release) (kube_pod_container_status_running{container=~".*(scheduler|scheduler-gc|webserver|worker|statsd|pgbouncer|metrics-exporter|redis|flower)"}) - count by(release) (kube_pod_container_status_running{container=~".*(scheduler|scheduler-gc|webserver|worker|statsd|pgbouncer|metrics-exporter|redis|flower)"}) < 0
+          expr: sum by(release) (kube_pod_container_status_running{container=~".*(scheduler|scheduler-gc|webserver|worker|statsd|pgbouncer|metrics-exporter|redis|flower|triggerer)"}) - count by(release) (kube_pod_container_status_running{container=~".*(scheduler|scheduler-gc|webserver|worker|statsd|pgbouncer|metrics-exporter|redis|flower|triggerer)"}) < 0
```

Comment:
alert was missing triggerer

Reply:
will this cause in issue when it doesn't have triggerer?

Reply:
no this is querying for these if it is not present that will be fine it just won't anything to the count as it should not

Reply:
cool just wanted to make sure